youtube_dl/extractor/nuevo.py

```diff
@@ -0,0 +1,33 @@
+# encoding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+from ..utils import float_or_none
+
+class NuevoBaseIE(InfoExtractor):
+    def _extract_nuevo(self, config_url, video_id, info=None, ignore_hd=False):
```

Comment:
@dstftw [Well, this obviously won't work.](https://github.com/rg3/youtube-dl/blob/master/youtube_dl/extractor/common.py#L837)

```python
    ...
    
    def _check_formats(self, formats, video_id):
        if formats:
            formats[:] = filter(
                lambda f: self._is_valid_url(
                    f['url'], video_id,
                    item='%s video format' % f.get('format_id') if f.get('format_id') else 'video'),
                formats)

    def _is_valid_url(self, url, video_id, item='video'):
        url = self._proto_relative_url(url, scheme='http:')
        # For now assume non HTTP(S) URLs always valid
        if not (url.startswith('http://') or url.startswith('https://')):
            return True
    
    ...
```